config/formats.js

```diff
@@ -404,7 +404,7 @@ exports.Formats = [
 				let abilityMap = Object.create(null);
 				for (let speciesid in Dex.data.Pokedex) {
 					let pokemon = Dex.data.Pokedex[speciesid];
-					if (pokemon.num < 1 || Dex.data.FormatsData[speciesid].tier === 'Uber' || this.format.banlist.includes(pokemon.species) || pokemon.species === 'Smeargle') continue;
+					if (pokemon.num < 1 || this.format.banlist.includes(pokemon.species) || pokemon.species === 'Smeargle') continue;
```

Comment:
By that token could you remove the banlist check, since that would fail the validation later anyway? Or would it actually be better to leave this as-is from an efficiency viewpoint (no point putting Ubers in the ability map)?

Reply:
I personally think it's just better to leave this check in for efficiency, just removing the specific part that I did is because it's redundant.

Reply:
I recommend removing this. `.banlist` is not supposed to be read directly ever.